packages/ui-kit-components/src/components/IndexTickers/IndexTickers.tsx

```diff
@@ -0,0 +1,118 @@
+import { Box, Divider, Palette, Typography, useTheme } from '@mui/material';
+import * as React from 'react';
+
+import * as Icons from '../../icons';
+
+export interface Props {
+  name: string;
+  latestPrice: number;
+  changedPrice: number;
+  percentage: number;
+  includeDivider?: boolean;
+  screen?: 'mobile' | 'desktop';
+}
+const getRoundedToTwo = (num: number) => {
+  return Number(num).toFixed(2).padEnd(2, '0');
+};
+const color = (palette: Palette, changedPrice: number) => {
+  if (changedPrice === 0) {
+    return {};
+  } else if (changedPrice < 0) {
+    return { color: palette.sell.main };
+  } else {
+    return { color: palette.success.main };
+  }
+};
+const IndexTickers: React.FC<Props> = ({
+  name = 'Name',
```

Comment:
Might be worth considering if we should have 0 for most of these values in case there's an issue when the data is fetched on a live app.

Reply:
I propose to use the `NA` or `-` as the default value for the different prices.

Reply:
I think `-` would be good.  Might be good to verify with Nate in case he has other plans for if there's an issue with the data for the ticker.